Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.mm

```diff
@@ -60,6 +60,32 @@
 
 using namespace WebCore;
 
+constexpr Seconds delegatedContentsFinishedTimeout = 5_s;
+
+namespace {
+
+class DelegatedContentsFenceFlusher final : public ThreadSafeImageBufferFlusher {
+public:
+    static std::unique_ptr<DelegatedContentsFenceFlusher> create(RefPtr<PlatformCALayerDelegatedContentsFence> fence)
+    {
+        return std::unique_ptr<DelegatedContentsFenceFlusher> { new DelegatedContentsFenceFlusher(WTFMove(fence)) };
```

Comment:
nit: std::make_unique

Reply:
It doesn't work because the constructor is private, using it will cause more problems than what it solves (it doesn't really solve anything..)